x-pack/test/api_integration/apis/search/search.ts

```diff
@@ -6,17 +6,53 @@
  */
 
 import expect from '@kbn/expect';
+import type { Context } from 'mocha';
 import { FtrProviderContext } from '../../ftr_provider_context';
 import { verifyErrorResponse } from '../../../../../test/api_integration/apis/search/verify_error';
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
+  const es = getService('es');
+  const log = getService('log');
+  const retry = getService('retry');
+
+  const shardDelayAgg = (delay: string) => ({
+    aggs: {
+      delay: {
+        shard_delay: {
```

Comment:
Using dev only shard_delay agg to stabilize checks the depend on how much time it took to get the result